lib/scoped_search/validators.rb

```diff
@@ -0,0 +1,7 @@
+module ScopedSearch
+  # This class will be used to store standard validators.
+  class Validators
+    NUMERIC = ->(value) { value =~ ScopedSearch::Definition.NUMERICAL_REGXP }
```

Comment:
Should be `::NUMERICAL_REGXP` as it's not a method on Definition.

(There's no test coverage here, I only found it didn't work by manual testing.)